nfiles.lisp

```diff
@@ -300,14 +300,44 @@ See `expand' for a convenience wrapper."))
       path
       (funcall xdg-fun path)))
 
+(defun flatpak-xdg-config-home (&rest more)
+  "Based on `uiop:xdg-config-home'."
+  (uiop:resolve-absolute-location
+   `(,(or (uiop:getenv-absolute-directory "HOST_XDG_CONFIG_HOME")
+          (uiop:subpathname (user-homedir-pathname) ".config/"))
+     ,more)))
+
 (defmethod resolve ((profile profile) (file config-file))
-  (maybe-xdg #'uiop:xdg-config-home (call-next-method)))
+  #-flatpak
```

Comment:
You could just move this branch to line 303 above instead, that way you don't have to put it everywhere.

Reply:
Yet another alternative: put them all inside a single `progn` and only use feature expression on this `progn`. That [still counts as top level form](https://cl-community-spec.github.io/pages/Processing-of-Top-Level-Forms.html), while being much easier to include/exclude 😃